browser/brave_rewards/sources.gni

```diff
@@ -72,6 +73,8 @@ if (is_android) {
   brave_browser_brave_rewards_deps += [ "//brave/components/brave_ads/browser" ]
 } else {
   brave_browser_brave_rewards_sources += [
+    "//brave/browser/brave_rewards/rewards_panel_coordinator.cc",
```

Comment:
we should not add new files to sources unless they have circular dependency issues. I didn't notice anything, but maybe I missed something. If there are no circular dependencies please put these in brave/browser/brave_rewards/rewards_panel and create a new BUILD.gn for them (applies to extension as well)

Reply:
Unfortunately, `rewards_panel_extension_handler` includes `rewards_service_factory.h`, which creates that circular dependency.  🤔

Reply:
can we pass the rewards service into the constructor (same with extension service)? That is preferred in general to passing in BrowserContext/Profile to reduce dependencies

Reply:
This suggestion worked well - now we no longer need the new context util (although I can imagine wanting to add it later).